using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-event-reference.md

```diff
@@ -51,22 +51,20 @@ The `trackNotificationReceive` parameters are also sent in the event, as describ
 
 The `trackNotificationClick` parameters are also sent in the event, as described below:
 
-```text
+```json
 {
-    "trackclick": <Boolean>
-    "trackinfo": <StringMap>
+    "trackclick": "<Boolean>",
+    "trackinfo": "<StringMap>"
 }
 ```
 
-### Configuration Response Content   <a id="configuration-response-content"></a>
+### Configuration response
 
 The data property in this event is used by each extension to modify its settings. Each extension is responsible for reading out the part of the data property for which it is concerned.
 
-For more information on the data payload definition in this event, see the [Configuration Keys](./#configuration-keys) section.
+#### Data payload definition 
 
-#### Data payload definition   <a id="data-payload-definition-2"></a>
-
-The Campaign Classic extension also reads the following key from the configuration event:
+The following key/value pairs are used by the configuration event in the Adobe Campaign Classic extension.
```

Comment:
The following key/value pairs are used by the Adobe Campaign Classic extension from the configuration event.